.drone.yml

```diff
@@ -0,0 +1,16 @@
+pipeline:
+  test:
+    image: scalacenter/scala-native:0.1.0
+    commands:
+      - export SBT_OPTS="-Xmx8G -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true"
+      - time nix-shell bin/scala-native.nix -A clangEnv --run "sbt rebuild test-all"
```

Comment:
Unfortunately, testing using nix gives me zero information if the code will work in the typical end-user environment like Ubuntu, given how special things are in that environment. 

Considering that it fails tests for reasons we don't know yet, I do not think it's a good time investment to use for CI in the first place. We don't have time budget to spend on debugging things in peculiar environments while something as common as Windows is still not supported.